.github/CODEOWNERS

```diff
@@ -142,13 +144,15 @@ packages/react-internal/src/components/ResizeGroup/ @micahgodbolt
 packages/react-internal/src/components/SearchBox/ @ecraig12345
 packages/react-internal/src/components/Separator/ @joschemd @khmakoto
 packages/react-internal/src/components/Shimmer/ @Vitalius1
+packages/react-internal/src/components/Slider/ @joschect
```

Comment:
Missing an entry for Pivot?

Reply:
We just never removed it, it's there on line 139.